templates/partials/answers/textfield.html

```diff
@@ -42,7 +42,8 @@
     "ariaOneResult": "There is one suggestion available.",
     "ariaNResults": "There are {n} suggestions available.",
     "ariaLimitedResults": "Results have been limited to 10 suggestions. Type more characters to improve your search.",
-    "autosuggestData": answer.suggestions_url
+    "autosuggestData": answer.suggestions.url,
+    "allowMultiple": answer.suggestions.allow_multiple | lower if answer.suggestions.allow_multiple is defined,
```

Comment:
The comma is invalid as this is the last statement, Jinja2 is too nice and doesn't fail. Given the value will always be a boolean, "is defined" is not needed, and we could remove the "lower". Also, given we send "Undefined" if allow multiple isn't defined, I think an else clause should be added for completion.
```suggestion
    "allowMultiple": "true" if answer.suggestions.allow_multiple else "false"
```